### PR TITLE
Identification and first fixes of log overwriting issues.

### PR DESCRIPTION
**Pull request by EstherDarkish**
In a second PR, the identified but not fixed issues will be adressed.

### CODE REVIEW

File: src/main/java/org/gridsuite/modification/server/modifications/BatteryCreation.java [hunk 1]
```diff
@@ -219,8 +219,9 @@ private void createBatteryActivePowerControl(BatteryCreationInfos batteryCreatio
             } catch (PowsyblException e) {
                 activePowerRegulationReports.add(Report.builder()
                         .withKey("ActivePowerExtensionAddError")
-                        .withDefaultMessage("cannot add active power extension on battery with id=${id} : " + e.getMessage())
+                        .withDefaultMessage("cannot add active power extension on battery with id=${id} : ${message}")
                         .withValue("id", batteryCreationInfos.getEquipmentId())
+                        .withValue("message", e.getMessage())
```
Comment (dbraquart) [inline src/main/java/org/gridsuite/modification/server/modifications/BatteryCreation.java:224]: I think as soon as you have a parameter ({id} here) in the message, it is ok

File: src/main/java/org/gridsuite/modification/server/modifications/OperatingStatusModification.java [hunk 2]
```diff
@@ -151,9 +154,10 @@ private void applyEnergiseEquipmentEnd(Reporter subReporter, Identifiable<?> equ
 
         subReporter.report(Report.builder()
                 .withKey("energise" + equipmentType + "EndApplied")
-                .withDefaultMessage(equipmentType + " ${id} (id) : energise the side ${side} applied")
+                .withDefaultMessage("${equipmentType} ${id} (id) : energise the side ${side} applied")
                 .withValue("id", equipment.getId())
                 .withValue("side", side.name())
+                .withValue("equipmentType", equipmentType)
```
Comment (dbraquart) [inline src/main/java/org/gridsuite/modification/server/modifications/OperatingStatusModification.java:160]: would be better if the withValue statements follow the same left->right order

**Review by dbraquart — APPROVED** [review]
code: ok (minor remarks)
tests: ok (partial)

File: src/main/java/org/gridsuite/modification/server/modifications/BatteryCreation.java (hunk 1 above)
Comment (EstherDarkish) [reply src/main/java/org/gridsuite/modification/server/modifications/BatteryCreation.java:224]: Every variable needs to be in the .withValue clause

File: src/main/java/org/gridsuite/modification/server/modifications/OperatingStatusModification.java (hunk 2 above)
Comment (EstherDarkish) [reply src/main/java/org/gridsuite/modification/server/modifications/OperatingStatusModification.java]: Fixed

**Review by ghazwarhili — APPROVED** [review]
code => ok

**Pull request merged**